scene/gui/color_picker.h

```diff
@@ -117,6 +117,10 @@ class ColorPicker : public VBoxContainer {
 #endif
 
 	int current_slider_count = SLIDER_COUNT;
+	// TODO: Think about better name or a way to not use it at all
+	Vector2i hsv_keyboard_picker_cursor_position;
```

Comment:
A way to not use it is keeping the variable local in `_uv_input()`, which means calculating sin/cos again every time.

Reply:
The hack mentioned in another place is connected with issues with calculating sin/cos again every time. I tried it at the beginning and it's behaviour was strange - the cursor was not going in straight lines. I believe it might be caused by some precision loss while calculating this, or something like that.

I realized that even when you drag the cursor with mouse, the offset of it's center and the mouse cursor is changing, depending on how high or low on the circle you are. You can see that on this video:

https://github.com/user-attachments/assets/baade90a-bea1-48d4-bff2-f3b151e62b7f